frontend/src/Explorer/Util/String.js

```diff
@@ -9,3 +9,9 @@ exports.substituteImpl = function(str) {
     });
   };
 };
+
+exports.take = function (n) {
```

Comment:
Why not `take` of `Data.String` ? https://pursuit.purescript.org/packages/purescript-strings/3.0.0/docs/Data.String#v:take  It is support with `v.2.0.1` (PS `v0.10.x`) as well  https://github.com/purescript/purescript-strings/blob/v2.1.0/src/Data/String.purs#L220